packages/botonic-plugin-flow-builder/src/index.ts

```diff
@@ -51,10 +54,10 @@ export default class BotonicPluginFlowBuilder implements Plugin {
   }
 
   async readFlowContent(): Promise<FlowBuilderData> {
```

Comment:
Son we will have a change to be able to read for the latest published flow here using a keyword like this:
`.../flow/latest`
I will inform you!

Reply:
cool! let me know when it's ready, I'll open a separate PR